app-crypt/argon2/argon2-20160821.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Password hashing software that won the Password Hashing Competition (PHC)"
+HOMEPAGE=""https://github.com/P-H-C/phc-winner-argon2
+SRC_URI="https://github.com/P-H-C/phc-winner-argon2/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( Apache-2.0 CC0-1.0 )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="-static-libs"
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+S="${WORKDIR}/phc-winner-${P}"
+
+src_prepare() {
+	eapply "${FILESDIR}/${P}-makefile-install-target.patch"
+	eapply "${FILESDIR}/${P}-makefile-soname-symlinks.patch"
+	# Makefile creates static libs by default
+	use static-libs || sed -i -e 's/LIBRARIES = \$(LIB_SH) \$(LIB_ST)/LIBRARIES = \$(LIB_SH)/' Makefile || die "sed failed!"
```

Comment:
This is slightly misleading although syntactically correct. Could you slip everything after the first `||` into a subcontext like so? `use static-libs || { sed -i -e ... }`.